src/components/TodoList/TodoList.tsx

```diff
@@ -1,44 +1,99 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './TodoList.scss';
+import { Todo } from '../../react-app-env';
+import { getAllTodos } from '../../api';
 
-export const TodoList: React.FC = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
+interface Props {
+  selectedUserId: number,
+  chahgeSelectedUserId: (id: number) => void,
+}
+
+enum Options {
+  all,
+  active,
+  completed,
+}
+
+export const TodoList: React.FC<Props> = (props) => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [input, setInput] = useState('');
+  const [status, setStatus] = useState(Options.all);
+
+  useEffect(() => {
+    getAllTodos()
+      .then(response => setTodos(response));
```

Comment:
you have to handle errors, if getAllTodos return an error you should catch the error hrer